cascade/src/main/java/me/saket/cascade/CascadePopupMenu.kt

```diff
@@ -150,6 +153,18 @@ open class CascadePopupMenu @JvmOverloads constructor(
   fun dismiss() =
     popup.dismiss()
 
+  fun applyBackgroundDim(parent: View, @IntRange(from = 0, to = 255) dimAmount: Int) {
+    val dim: Drawable = ColorDrawable(Color.BLACK)
+    dim.setBounds(0, 0, parent.width, parent.height)
+    dim.alpha = dimAmount
+    val overlay = parent.overlay
+    overlay.add(dim)
```

Comment:
can the showing and hiding of this overlay be animated?

Reply:
Added animation for applying dim and removing.